open_schools_platform/student_management/students/services.py

```diff
@@ -91,6 +92,15 @@ def create_studentprofileicrcle_additional(text: str = None, parent_phone: Phone
     return additional
 
 
+def get_student_profile_or_create_new(student_phone: str, student_name: str):
+    student_profile = get_student_profile(filters={"phone": student_phone})
```

Comment:
we should search student_profile by family
give a better name to method

Reply:
(filters={"family": family.student_profiles}) like this?

Reply:
or get all student_profiles by family id?